custom/icds_reports/views.py

```diff
@@ -442,6 +442,10 @@ def query_month_in_range(self, query_month, start_month):
             in_range = False
         return in_range
 
+    @property
+    def bihar_state_id(self):
```

Comment:
can you call this function something like `get_state_id_from_state_name(state_name):`
this would let this base table not specific to just Bihar API.

Reply:
done